components/MarkdownEditor.tsx

```diff
@@ -0,0 +1,48 @@
+import { useSelector } from 'react-redux';
+import Editor from 'rich-markdown-editor';
+
+import { getTheme } from '../redux/selectors/settings.selectors';
+
+interface MarkdownEditorParams {
+  defaultValue: string;
+  placeholder: string;
+  onSave: () => void;
+  onChange: (description: string) => void;
+}
+
+import { dark, light } from 'rich-markdown-editor/dist/theme';
+
+const MarkdownEditor = ({
+  defaultValue,
+  placeholder,
+  onSave,
+  onChange,
+}: MarkdownEditorParams): JSX.Element => {
+  const globalTheme = useSelector(getTheme);
+  const modifiedDark = { ...dark, background: 'transparent' };
+  const markdownTheme = globalTheme === 'dark' ? modifiedDark : light;
+
+  //todo: figure out why we can't refresh
+
+  return (
+    <Editor
+      key={defaultValue}
+      defaultValue={defaultValue}
```

Comment:
After saving, the edit event is triggered but this default value is getting cached, todo: figure out how to go around it

Reply:
this seems fixed now